circle.yml

```diff
@@ -32,86 +32,39 @@ dependencies:
       make deps &&
       mkdir -p $(dirname $SRCDIR) &&
       cp -r $(pwd)/ $SRCDIR
-    - "cd $SRCDIR/client; ../tools/rebuild-image weaveworks/scope-ui-build . Dockerfile package.json webpack.production.config.js .eslintrc .babelrc && touch $SRCDIR/.scope_ui_build.uptodate"
-    - "cd $SRCDIR/backend; ../tools/rebuild-image weaveworks/scope-backend-build . Dockerfile build.sh && touch $SRCDIR/.scope_backend_build.uptodate"
+    - "cd $SRCDIR/client; ../tools/rebuild-image openebs/scope-ui-build . Dockerfile package.json webpack.production.config.js .eslintrc .babelrc && touch $SRCDIR/.scope_ui_build.uptodate"
+    - "cd $SRCDIR/backend; ../tools/rebuild-image openebs/scope-backend-build . Dockerfile build.sh && touch $SRCDIR/.scope_backend_build.uptodate"
     - test -z "$SECRET_PASSWORD" || (cd $SRCDIR/integration; ./gce.sh make_template):
         parallel: false
     - sudo apt-get update && sudo apt-get install python-pip && sudo pip install awscli
 
 test:
   override:
-    - cd $SRCDIR; make RM= lint:
```

Comment:
why is this removed? I think we should check linting errors before creating a build.